src/Common/mysqlxx/PoolWithFailover.cpp

```diff
@@ -160,6 +161,7 @@ PoolWithFailover::Entry PoolWithFailover::get()
                     }
 
                     app.logger().warning("Connection to " + pool->getDescription() + " failed: " + e.displayText());
+                    error_detail =  e.displayText();
```

Comment:
This error detail will bind only to one replica, if user tries to connect to 3 replicas and all of them will produce different errors it will be hard to understand which error belongs to which replica. I suggest to bind error_detail to replica, and then in result error output them together.
Also need to output MySQL error code check mysqld_error.h.

Reply:
thanks. I fixed and try to be consistent with MySQL error.